docs/configuration.md

```diff
@@ -40,26 +61,276 @@ cd src/CarbonAware.WebApi
 Logging__LogLevel__Default="Debug" dotnet run
 ```
 
-## CarbonAware.LocationSources
+## DataSources
+
+The SDK supports multiple data sources for getting carbon data.  At this time, only a JSON file and [WattTime](https://www.watttime.org/) are supported.
+Each data source interface is configured with a specific data source implementation.  
+
+If set to `WattTime`, WattTime configuration must also be supplied.
+
+`JSON` will result in the data being loaded from the file specified in the `DataFileLocation` property
+
+```json
+{
+"DataSources": {
+    "EmissionsDataSource": "Json",
+    "ForecastDataSource": "WattTime",
+    "Configurations": {
+      "WattTime": {
+        "Type": "WattTime",
+        "Username": "username",
+        "Password": "password",
+        "BaseURL": "https://api2.watttime.org/v2/",
+        "Proxy": {
+          "useProxy": true,
+          "url": "http://10.10.10.1",
+          "username": "proxyUsername",
+          "password": "proxyPassword"
+        }
+      },
+      "Json": {
+        "Type": "Json",
+        "DataFileLocation": "test-data.json"
```

Comment:
The file name needs to be changed to test-data-azure-emissions.json here and in the rest of the document.